pyracf/common/security_admin.py

```diff
@@ -54,6 +54,10 @@ class SecurityAdmin:
 
     _valid_segment_traits = {}
     _extracted_key_value_pair_segment_traits_map = {}
+    _racf_key_map = {}
+    _racf_message_key_map = {}
+    _racf_key_map = {}
+    _racf_message_key_map = {}
```

Comment:
These definitions are repeated. My guess this is a typo?

Reply:
yes, my apologies. Issue when debugging, no longer a problem.